modules/google-analytics/wp-google-analytics.php

```diff
@@ -70,6 +70,51 @@ public static function get_instance() {
 
 		return self::$instance;
 	}
+
+	/**
+	 * Add amp-analytics tags.
+	 *
+	 * @param array $analytics_entries An associative array of the analytics entries.
+	 *
+	 * @return array
+	 */
+	public static function amp_analytics_entries( $analytics_entries ) {
+		if ( ! is_array( $analytics_entries ) ) {
+			$analytics_entries = array();
+		}
+
+		$config_data = wp_json_encode(
+			array(
+				'vars'     => array(
+					'account' => Jetpack_Google_Analytics_Options::get_tracking_code(),
+				),
+				'triggers' => array(
+					'trackPageview' => array(
+						'on'      => 'visible',
+						'request' => 'pageview',
+					),
+				),
+			)
+		);
+
+		// Generate a hash string to uniquely identify this entry.
+		$entry_id = substr( md5( 'googleanalytics' . $config_data ), 0, 12 );
+
+		$data = array(
```

Comment:
Before we set this, we should check that the GA is not already set up by another source (i.e. You can set up Analytics using the AMP plugin or Google Sitekit and a number of other plugins). We should additionally check if the GA account is the same and if so, then bail. If it's different, we should add the tracking.

Reply:
This was addressed.